02-create.md

```diff
@@ -419,3 +419,5 @@ but it does find the copy in `thesis` that we didn't delete.
 > The command `ls -t` lists things by time of last change,
 > with most recently changed files or directories first.
 > In what order does `ls -R -t` display things?
+>
```

Comment:
How about itemizing the questions:

* In what order does ..
* How could we display ...